src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/response/SearchStaffUserResponse.java

```diff
@@ -17,7 +17,7 @@
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
-@Builder
+@SuperBuilder
```

Comment:
its not recommended to use experimental features, find alternative solution to achieve the same.